notebooks/cugraph_benchmarks/nx_cugraph_benchmark.ipynb

```diff
@@ -4,13 +4,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Benchmarking Performance of NetworkX with Rapids GPU-based nx_cugraph backend vs on cpu\n",
+    "# Benchmarking Performance of NetworkX with and without the RAPIDS GPU-based nx_cugraph backend\n",
```

Comment:
nx_cugraph? We should try to come up with consistent naming scheme. Here, nx-cugraph would seem better to me.

Reply:
Yeah I agree, that was an oversight, thanks for finding that.  I've been trying to apply it like this:
`nx-cugraph` : the name of the project and the package to install
`nx_cugraph` : used only in code
`nx-cuGraph` : never used